src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -1,36 +1,39 @@
 package org.apache.mesos.hdfs.util;
 
-public class HDFSConstants {
+/**
+ * defines the constants shared between the scheduler and the executors
+ */
+public interface HDFSConstants {
```

Comment:
2 reasons
1. because a class defines an implementation for an object and there is no implementation here
2. because is fairly standard if you are defining constants. similar to http://docs.oracle.com/javase/6/docs/api/javax/swing/SwingConstants.html

I can make it a final class if you prefer.